target/linux/ipq806x/config-5.4

```diff
@@ -78,6 +78,7 @@ CONFIG_CC_HAS_KASAN_GENERIC=y
 CONFIG_CLKDEV_LOOKUP=y
 CONFIG_CLKSRC_QCOM=y
 CONFIG_CLONE_BACKWARDS=y
+CONFIG_CMDLINE_OVERRIDE=y
```

Comment:
Why is this neccessary?

Reply:
Yes. Original command line is:
```[    0.000000] Kernel command line: console=ttyHSL1,115200n8 ubi.mtd=rootfs root=mtd:ubi_rootfs rootfstype=squashfs```

I use smem dynamic partition layout and OpenWrt method to find ubi rootfs, so I need to clear all info about rootfs in bootargs.

Reply:
Ok, makes sense if smem is used.
Otherwise, simple append could be used